cpp/SConstruct

```diff
@@ -39,8 +50,9 @@ To use emscripten to compile the library to asm.js, run:
   scons carfac.js
 
 To install emscripten and its dependencies on Ubuntu, follow the
-instructions on
-https://github.com/kripken/emscripten/wiki/Getting-Started-on-Ubuntu-12.10
+instructions on the emscriptenv website
+https://emscripten.org/docs/getting_started/downloads.html
```

Comment:
I just did `sudo aptitude install emscripten`. Why did you recommend using apt for the other deps, and the upstream installation instructions for emscripten?

Reply:
Because the specific document referenced originally at line 43 of master no-longer exists, and I cannot verify why there would have been specific instructions about onboarding for Ubuntu.
The Emscripten developers do not recommend using apt in their own documentation, they direct users to build from source, including the use of apt to install some dependencies. They explicitly say in the docs 

> Emsdk does not install any tools to the system, or otherwise interact with Linux package managers. All file changes are done inside the emsdk/ directory.

So I cannot assume that any package-maintainers version is either totally legitimate or up to date.

Reply:
Fair enough, I have no issues with referring to the upstream installation instructions.

Reply:
thank you